modules/microshift-cni-multus-bridge-nw-example-config.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-cni-multus-bridge-nw-example-config_{context}"]
+= Configuring a bridge-type additional network
+
+After you have created the NetworkAttachmentDefinition object and applied it, use the following example procedure to configure a bridge-type additional network. You can also use this workflow for other additional network types.
```

Comment:
I see this also is the same workflow for all types. Same suggestion for that as earlier. Alternatively, maybe something up higher in the assembly that covers all these topics off the bat, like "the following procedures show the `bridge` type as an example but apply to all interface types" so the reader who is not using `bridge` knows to pay attention?